azure-pipelines.yml

```diff
@@ -89,7 +89,7 @@ jobs:
   - script: |
       git clone https://github.com/oneapi-src/level-zero.git
       cd level-zero
-      git checkout v1.5.0
+      git checkout v1.6.2
```

Comment:
Is there a reason to not use 1.7.15?

Reply:
1.7.15 supports the 1.3.7 version of the spec right? And, as of now, no released version of libze supports the 1.4 version of the spec?

Yes, we can bump it to 1.7.15

Reply:
1.7.15 is a loader for implementation of 1.3.x spec as best I understand.

Reply:
Just checked the `apt` repo is right now on 1.7.9. So going to bleeding edge will result in having to build L0 or installing packages by hand. I want to avoid that to simplify setup as much as possible. Might as well leave it as is for now.